### ISSUE TITLE
Tabs: keyboard a11y is not consistent with MIND patterns

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/ebayui-core#422]
# Bug Report

## Description

When navigating tabs with arrow keys, tab panel should change automatically. 

https://ebay.gitbooks.io/mindpatterns/content/disclosure/tabs.html

## Workaround

Pressing SPACEBAR on a tab will change the tab panel.

**Comment by mkchang** [comment]
per Ian, there was a WCAG update https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel that now needs to be taken into account:
```
It is recommended that tabs activate automatically when they receive focus as long as their associated tab panels are displayed without noticeable latency. This typically requires tab panel content to be preloaded. Otherwise, automatic activation slows focus movement, which significantly hampers users' ability to navigate efficiently across the tab list. For additional guidance, see 5.4 Deciding When to Make Selection Automatically Follow Focus.
```
> two models of activation: auto (left/right arrows) and manual (left/right then space)

**Comment by ianmcburnie** [comment]
Merged to 7.0.0
